examples/bzlmod/tests/BUILD.bazel

```diff
@@ -48,6 +48,12 @@ py_test(
     deps = ["//libs/my_lib"],
 )
 
+py_test(
```

Comment:
I observed that the COVERAGE_RC env is not available for `py_test_3_9` et.al. 
Is this expected or is there an issue with the way the module is being registered

Reply:
You can debug this with `--toolchain_resolution_debug=<name of the toolchain>`. I don't think this is expected.

Reply:
the issue was that the rule transitions was not forwarding the env variables.